pahfit/model.py

```diff
@@ -734,158 +758,169 @@ def tabulate(
             # any other iterable will be accepted and converted to array
             wav = np.asarray(wavelengths) * u.micron
 
-        # shift the "observed wavelength grid" to "physical wavelength grid"
-        wav /= 1 + redshift
-        flux_values = flux_function(wav.value)
+        # apply feature mask, make sub model, and set up functional
+        if feature_mask is not None:
+            features_copy = self.features.copy()
```

Comment:
This seems unnecessary, as indexing already creates a new `Features` view.

Reply:
More precisely, basic slicing returns a view, while advanced indexing returns a copy. I did a copy() as a safeguard against unwanted modifications. Might not be necessary after all. Removing copy for now.